lapce-core/queries/prisma/highlights.scm

```diff
@@ -0,0 +1,56 @@
+(string) @string
+
+(enumeral) @constant
+(number) @constant.numeric
+
+(variable) @variable
```

Comment:
Where is the file from?

Reply:
here: [https://github.com/victorhqc/tree-sitter-prisma/blob/master/queries/highlights.scm](https://github.com/victorhqc/tree-sitter-prisma/blob/master/queries/highlights.scm)

Reply:
Please include that information like other queries do

Reply:
Oh, it should be written in the head of the file, right?

Reply:
OK, it has been added. I misunderstood